scripts/install2

```diff
@@ -226,6 +226,9 @@ fi
 if ! [ -x "/opt/cjdns/cjdroute" ]; then
     here=`pwd`
     cd /opt/cjdns && eval $CJDNS_BUILD_CMD && cd $here
+    if ! [ -x "/usr/bin/cjdroute" ]; then
+        sudo rm -f /usr/bin/cjdroute
```

Comment:
Can we close this PR and make a single commit that puts `sudo rm -f /usr/bin/cjdroute` before `sudo cp /opt/cjdns/cjdroute /usr/bin/cjdroute`, like this:

```
# Install cjdns to /usr/bin
if ! [ -x "/usr/bin/cjdroute" ]; then
    sudo rm -f /usr/bin/cjdroute
    sudo cp /opt/cjdns/cjdroute /usr/bin/cjdroute
fi
```

Because the removal is part of install and not build. I think this is most clear. And new commit as all the history here are a little confusing.